### ISSUE TITLE
Is \ingroup not supported?

### ISSUE THREAD

**Issue by dhanjit** [mosra/m.css#50]
I am facing troubles:

```
INFO:root:group__fire__and__forget__orders__group_1details_send_fire_and_forget_order_with_quote_and_side
Traceback (most recent call last):
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 3027, in <module>
    run(doxyfile, os.path.abspath(args.templates), args.wildcard, args.index_pages, search_merge_subtrees=not args.search_no_subtree_merging, search_add_lookahead_barriers=not args.search_no_lookahead_barriers, search_merge_prefixes=not args.search_no_prefix_merging)
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 2941, in run
    parsed = parse_xml(state, file)
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 2318, in parse_xml
    func = parse_func(state, memberdef)
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 1581, in parse_func
    func.description, templates, params, func.return_value, func.return_values, func.exceptions, search_keywords, func.is_deprecated = parse_func_desc(state, element)
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 1428, in parse_func_desc
    parsed = parse_desc_internal(state, element.find('detaileddescription'))
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 777, in parse_desc_internal
    add_css_class=add_css_class)
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 1237, in parse_desc_internal
    out.parsed += '<a name="{}"></a>'.format(extract_id_hash(state, i))
  File "/home/dhanjit.d/dump/m.css/doxygen/dox2html5.py", line 424, in extract_id_hash
    assert id.startswith(state.current_compound.url_base)
AssertionError
python3.6 ~/dump/m.css/doxygen/dox2html5.py Doxyfile-mcss --debug  111.52s user 3.01s system 99% cpu 1:54.63 total
```

**Comment by dhanjit** [comment]
```
    /** Update status for the strategy.

        This function is called on the ui status update.
        \ingroup list_of_stop_conditions
        \param[in] status_update bool representing value from ui. \c true means start, \c false means stop.
        \anchor details_updatestatus \link details_updatestatus More details \endlink
     */
```

IF I remove the \anchor details part it works, if i remove the \ingroup part, then also it works.

**Comment by mosra** [comment]
Hi, thanks for the repro case, that really helped me narrow it down. Should be fixed in ac1da96a7b2969c36b86daaa5b281e5e7608d673 now.
